### DISCUSSION TITLE
Does artisan make:filament-role exist? How do I generate it?

### DISCUSSION THREAD

**Discussion by ddoddsr** [filamentphp/filament#6279] [Help]
Following the documentation for securing a custom page, I ran  `php artisan make:filament-role Manager ` and got   ` ERROR  Command "make:filament-role" is not defined. Did you mean one of these? ` 

After checking the [documentation ](https://filamentphp.com/docs/1.x/admin/pages#authorization), spelling and typing there must be something I'm missing or the document requires updating.

**Comment by danharrin** [comment]
That was a v1 feature, you're on the wrong version of the documentation. You should use a plugin for authorisation in v2 like Shield.

**Comment by ddoddsr** [comment]
My bad , not checking the version. thanks!
I think I can work out how to use  shouldRegisterNavigation() and mount() to get what I need.
I am trying to make shield work with my pages.
